src/components/Terminal/setup-terminal.js

```diff
@@ -0,0 +1,98 @@
+import { Terminal as Xterm } from "xterm";
+import { FitAddon as XtermFitAddon } from "xterm-addon-fit";
+
+import XtermTerminadoAddon from "./xterm-addon-terminado";
+
+// Define jujushell API operations and codes.
+const OP_LOGIN = "login";
+const OP_START = "start";
+const CODE_OK = "ok"; // eslint-disable-line no-unused-vars
```

Comment:
ok

Reply:
Hah, yeah this will be used in a follow-up branch but just defining it here so I don't forget :)